client/hwlib/build.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    let os_release_file_path =
+        std::env::var("OS_RELEASE_FILE_PATH").unwrap_or("/etc/os-release".to_string());
```

Comment:
Small nitpick here, the `.to_string()` method comes from the `Display` trait which allows for arbitrary, pretty re-formatting. I think the intention here isn't pretty-printing, but rather to change the ownership model from borrowed (`&str`) to owned (`String`), hence using the `.to_owned()` method would be slightly better to signal the intention